pkg/internal/block_device.go

```diff
@@ -89,42 +86,17 @@ func ListBlockDevices(exec Executor) ([]BlockDevice, error) {
 	return blockDeviceMap["blockdevices"], nil
 }
 
-func ListDiskByPath(exec Executor) (map[string]string, error) {
```

Comment:
Is this a cleanup of something that's not used?

Reply:
Yes, this was a leftover code that is not used anywhere so I deleted it.